n3fit/runcards/example-nnpdf41.yml

```diff
@@ -0,0 +1,188 @@
+- {dataset: CMS_TTBAR_13TEV_LJ_DIF_YT, frac: 0.75}
+- {dataset: CMS_SINGLETOP_7TEV_TCHANNEL-XSEC, frac: 0.75}
+- {dataset: CMS_SINGLETOP_8TEV_TCHANNEL-XSEC, frac: 0.75}
+- {dataset: CMS_SINGLETOP_13TEV_TCHANNEL-XSEC, frac: 0.75}
+- {dataset: LHCB_Z0_7TEV_DIELECTRON_Y, frac: 0.75}
+- {dataset: LHCB_Z0_8TEV_DIELECTRON_Y, frac: 0.75}
+- {dataset: LHCB_WPWM_7TEV_MUON_Y, frac: 0.75, cfac: [NRM]}
+- {dataset: LHCB_Z0_7TEV_MUON_Y, frac: 0.75, cfac: [NRM]}
+- {dataset: LHCB_WPWM_8TEV_MUON_Y, frac: 0.75, cfac: [NRM]}
+- {dataset: LHCB_Z0_8TEV_MUON_Y, frac: 0.75, cfac: [NRM]}
+- {dataset: LHCB_Z0_13TEV_DIMUON-Y, frac: 0.75}
+- {dataset: LHCB_Z0_13TEV_DIELECTRON-Y, frac: 0.75}
+
+
+################################################################################
+datacuts:
+  t0pdfset: 250917-jcm-001
+  q2min: 3.49
+  w2min: 12.5
+theory:
+  theoryid: 41_000_000
+
+trvlseed: 1953065998
+nnseed: 1589400026
+mcseed: 2135943670
+genrep: true
+parameters: # This defines the parameter dictionary that is passed to the Model Trainer
+  nodes_per_layer: [70, 50, 25, 20, 9]
+  activation_per_layer: [tanh, tanh, tanh, tanh, linear]
+  initializer: glorot_normal
+  optimizer:
+    clipnorm: 6.073e-6
+    learning_rate: 2.621e-3
+    optimizer_name: Nadam
+  epochs: 27000
+  positivity:
+    initial: 184.8
+    multiplier:
+  integrability:
+    initial: 10
+    multiplier:
+  stopping_patience: 0.1
+  layer_type: dense
+  dropout: 0.0
+  threshold_chi2: 3.5
+  interpolation_points: 5
+fitting:
+  fitbasis: CCBAR_ASYMM  # EVOL (7), EVOLQED (8), etc.
+  savepseudodata: true
+  basis:
+  - {fl: sng, trainable: false, smallx: [1.103, 1.12], largex: [1.461, 3.778]}
+  - {fl: g, trainable: false, smallx: [0.8737, 1.088], largex: [2.204, 4.431]}
+  - {fl: v, trainable: false, smallx: [0.5051, 0.6847], largex: [1.526, 2.557]}
+  - {fl: v3, trainable: false, smallx: [0.1932, 0.4309], largex: [1.73, 2.559]}
+  - {fl: v8, trainable: false, smallx: [0.5645, 0.7228], largex: [1.566, 2.662]}
+  - {fl: t3, trainable: false, smallx: [-0.4244, 1.0], largex: [1.75, 2.939]}
+  - {fl: t8, trainable: false, smallx: [0.6677, 0.9282], largex: [1.55, 3.504]}
+  - {fl: t15, trainable: false, smallx: [1.087, 1.136], largex: [1.503, 3.379]}
+  - {fl: v15, trainable: false, smallx: [0.4713, 0.7641], largex: [1.464, 3.851]}
+
+################################################################################
+positivity:
+  posdatasets:
+  # Positivity Lagrange Multiplier
+  - {dataset: NNPDF_POS_2P24GEV_F2U, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_F2D, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_F2S, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_FLL, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_DYU, maxlambda: 1e10}
+  - {dataset: NNPDF_POS_2P24GEV_DYD, maxlambda: 1e10}
+  - {dataset: NNPDF_POS_2P24GEV_DYS, maxlambda: 1e10}
+  - {dataset: NNPDF_POS_2P24GEV_F2C-CCE-17PTS, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_F2C-CCP-17PTS, maxlambda: 1e6}
```

Comment:
I guess these points ill be cut with any choice that's going to be made, but if you want to get rid of the -17PTS now is probably a good time. 

The 40_ theories are a bit of a mix between -17PTS and not, which is something to avoid since it means you have to check which positivity names exist in the theory before launching a job